recipes-devtools/libfmi/libfmi/0001-fmixml-use-system-expat-instead-of-building-own-with.patch

```diff
@@ -0,0 +1,126 @@
+From e0ab696a4d304e99a18669354077872258a9909b Mon Sep 17 00:00:00 2001
+From: Martin Jansa <martin.jansa@lge.com>
+Date: Tue, 9 Jun 2020 12:48:53 +0000
+Subject: [PATCH] fmixml: use system expat instead of building own with
+ ExternalProject
+
+it generates duplicate rules for ExpatEx/libexpat.a and newer ninja-1.9.0 fails because of that:
+NOTE: VERBOSE=1 cmake --build fmilibrary/2.0.3-r0/build --target all -- -j 20
+ninja: error: build.ninja:1652: multiple rules generate ExpatEx/libexpat.a [-w dupbuild=err]
+
+Upstream-Status: Pending
```

Comment:
This `.patch` was authored in June of 2020. Is it still actually `pending` upstream? Or can we bump our libfmi version and drop the .patch?

Reply:
I don't see it in their `master` or `dev_3.0` trees and it still appears to be necessary as of 2.3 (the latest release; May 2021).